packages/story-editor/src/utils/usePageAsCanvas.js

```diff
@@ -27,7 +27,7 @@ import { ZOOM_SETTING } from '../constants';
 import { getAccessibleTextColorsFromPixels } from './contrastUtils';
 import { calculateTextHeight } from './textMeasurements';
 
-function usePageAsCanvas() {
+function usePageAsCanvas(element) {
```

Comment:
We should probably rename this to something more clear, e.g. `toExclude` or `elementToExclude`.

Alternatively we could make this a flag instead, e.g. `excludeSelectedElement` and then we could get the selected element here instead.

Reply:
Currently the feature doesn't support multi-selection, so there's no clear way how to account for multi-selection since due to the same question of if to consider each text as separate or all together, it was decided not to support multi-selection (for now).